### ISSUE TITLE
External-link.js: If link contains img, get alt tag content to aria-label

### ISSUE THREAD

**Issue by ronilaukkarinen** [digitoimistodude/air-light#111]
For example:

``` html
<a href="https://www.google.com">
  <img src="https://www.google.fi/images/branding/googlelogo/2x/googlelogo_color_160x56dp.png" alt="Google" />
</a>
```

This is currently outputting as:

``` html
<a href="https://www.google.com" aria-label="External site: " class="is-external-link">
  <img src="https://www.google.fi/images/branding/googlelogo/2x/googlelogo_color_160x56dp.png" alt="Google" />
</a>
```

It should be like this:

``` html
<a href="https://www.google.com" aria-label="External site: Google" class="is-external-link">
  <img src="https://www.google.fi/images/branding/googlelogo/2x/googlelogo_color_160x56dp.png" alt="Google" />
</a>
```

If there's target="_blank", it should be:

``` html
<a href="https://www.google.com" aria-label="External site: Google, opens in a new window" class="is-external-link" target="_blank">
  <img src="https://www.google.fi/images/branding/googlelogo/2x/googlelogo_color_160x56dp.png" alt="Google" />
</a>
```

Changes needed in [external-link.js](https://github.com/digitoimistodude/air-light/blob/1a025faad3a7b730f0e742ca3aee3c1c5b3b16e5/js/src/modules/external-link.js) for it to be able to recognize the link type and parse the alt.
